setup.py

```diff
@@ -13,15 +10,17 @@
     from buildbot_pkg import setup_www_plugin
 except ImportError:
     import sys
-    print("Please install buildbot_pkg module in order to install that package, or use the pre-build .whl modules available on pypi", file=sys.stderr)
+    print('Please install buildbot_pkg module in order to install that '
+          'package, or use the pre-build .whl modules available on pypi',
+          file=sys.stderr)
     sys.exit(1)
 
 setup_www_plugin(
     name='buildbot-macports-custom-views',
     description='MacPorts Custom Views Plugin for Buildbot',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    version='0.0.2',
+    version='0.0.3',
```

Comment:
lets say its version 1.0!

Reply:
Ok

Reply:
Well, ... I wouldn't say it's quite 1.0 yet.